### ISSUE TITLE
[REQUEST] Upload documentation for new SM-031 (Zigbee module) by CoolKit Technologies

### ISSUE THREAD

**Issue by Hedda** [CoolKit-Technologies/DevDocs#12]
@junlin-chen @june0854 @coolkit-carl Request you please upload documentation for your new SM-031 Zigbee module:

https://github.com/CoolKit-Technologies/DevDocs/tree/master/Zigbee

This new SM-031 (Zigbee module) module by @CoolKit-Technologies is used in SONOFF ZBBridge Zigbee Bridge Pro (ZBBridge-P) by ITead and GPIO pin documentation is requested by the Tasmota (and ESPHome) open source communities:

https://github.com/arendst/Tasmota/discussions/14419

https://itead.cc/product/sonoff-zigbee-bridge-pro/

https://fccid.io/2APN5ZBBRIDGEP

![image](https://user-images.githubusercontent.com/6320001/169269823-291a40e5-2db4-45c4-80f2-2f1cf003bd70.png)

![image](https://user-images.githubusercontent.com/6320001/169269920-b9277b55-0720-435e-bf59-a09fa5399878.png)

**Comment by divadiow** [comment]
I guess this didn't happen. I don't see any datasheets for it on GH

**Comment by Hedda** [comment]
No, but there is already support in Tasmota so they reverse engineered it:

* https://github.com/arendst/Tasmota/discussions/14419

* https://tasmota.github.io/docs/Zigbee/

* https://zigbee.blakadder.com/Sonoff_ZBBridge-P.html

* https://templates.blakadder.com/sonoff_ZBBridge-P.html

So if have further questions suggest posting in their community instead:

* https://github.com/arendst/Tasmota/discussions/
